.github/CONTRIBUTING.md

```diff
@@ -216,3 +222,24 @@ from Transifex to the source code and vice versa.
 ### Translations (for translators) ###
 
 Please have a look at the specific [FAQ for translators](https://github.com/Cockatrice/Cockatrice/wiki/Translation-FAQ).
+
+
+# Release Management #
+
+### ToDo list around new main releases ###
+
+ - write a `Release Draft` on github with all changes since last main release
+ - build newest strings and upload them to transifex
+ - give translators some time --> pull newest translations from transifex and PR them back in
+ - **Final commit:** push a PR which adds the `Release Name` and `Version Number` in the `About` window
```

Comment:
@Daenyth Actually with https://github.com/Cockatrice/Cockatrice/pull/2570 (outlined in https://github.com/Cockatrice/Cockatrice/pull/2613) this seems to be even more complex now :D
But I think that's how we have to do it?

Just realized while I wanted to update this pr...

cc: @ctrlaltca